Engines/Wine/Verbs/Windows XP SP 3/script.js

```diff
@@ -38,3 +39,22 @@ Wine.prototype.sp3extract = function (fileToExtract) {
 
     return this;
 };
+
+/**
+ * Verb to install Windows XP Service Pack 3
+*/
+var verbImplementation = {
```

Comment:
See above

Reply:
I'm not so sure about the functionality of this verb myself. I commented to my best guess. In any case, I think this is pretty much obsolete (Windows XP...).